tembo-cli/tests/tomls/minimal/tembo.toml

```diff
@@ -0,0 +1,3 @@
+[minimal]
+environment = "prod"
+instance_name = "minimal"
```

Comment:
In this PR, I'm seeking to get this toml to work and add docker test of it to CI

Reply:
This is awesome! Thank you.